staging/src/k8s.io/apiserver/pkg/registry/rest/delete.go

```diff
@@ -39,6 +39,7 @@ type GarbageCollectionPolicy string
 const (
 	DeleteDependents GarbageCollectionPolicy = "DeleteDependents"
 	OrphanDependents GarbageCollectionPolicy = "OrphanDependents"
+	Unsupported      GarbageCollectionPolicy = "Unsupported"
```

Comment:
I see. All "unsupported" resources should also be in the "ignored" list of GC. Could you say that explicitly in the comment of [Unsupported](https://github.com/kubernetes/kubernetes/pull/48354/files#diff-8efe9b63ce664799846ae9d995260e95R44)? The system will behave strangely if the "two sets of ignores" don't match.